neural_compressor/common/utils/utility.py

```diff
@@ -206,14 +206,31 @@ def set_tensorboard(tensorboard: bool):
 default_tuning_logger = TuningLogger()
 
 
-def log_quant_execution(func):
-    def wrapper(*args, **kwargs):
-        default_tuning_logger.quantization_start(stacklevel=4)
-
-        # Call the original function
-        result = func(*args, **kwargs)
-
-        default_tuning_logger.quantization_end(stacklevel=4)
-        return result
-
-    return wrapper
+def log_quant_execution(mode="quantize"):
+    def log_quant_execution_wrapper(func):
+        def inner_wrapper(*args, **kwargs):
+            start_log, end_log = None, None
+            if mode == "quantize":
```

Comment:
nit: To prevent the silent bug caused by a spelling error, please pass `Mode.xxx` (for example, `Mode.QUANTIZE`) to `mode` instead of `Mode.QUANTIZE.value`, and compare `mode` with `Mode.xxx`. This static checker will pinpoint spelling errors.

Reply:
If so, will move `neural_compressor.torch.utils.Mode` to `common` dir, but it is only used for torch framework.